rs/bitcoin/checker/src/main.rs

```diff
@@ -88,13 +89,27 @@ fn check_address(args: CheckAddressArgs) -> CheckAddressResponse {
 /// fails to decode or its transaction id does not match, then `Error` is returned
 /// together with a text description.
 async fn check_transaction(args: CheckTransactionArgs) -> CheckTransactionResponse {
```

Comment:
supra nit: I would put any annotation (`#[ic_cdk::update]`) directly above the method signature, since usually the Rust docs apply to the whole (annotation+method signature)

Reply:
Good idea! Moved the annotation next to the signature.